sql/core/src/main/scala/org/apache/spark/sql/jdbc/JdbcDialects.scala

```diff
@@ -295,7 +295,6 @@ abstract class JdbcDialect extends Serializable with Logging{
    * Build a create index SQL statement.
    *
    * @param indexName         the name of the index to be created
-   * @param indexType         the type of the index to be created
```

Comment:
@huaxingao  The difference from #34486 is only this part right?

Reply:
Right. Sorry I forgot to delete this last time.